homeassistant/components/switch/broadlink.py

```diff
@@ -21,7 +21,10 @@
                                  CONF_TYPE)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['broadlink==0.2']
+
+REQUIREMENTS = ['https://github.com/mjg59/python-broadlink/archive'
```

Comment:
I am actually confused why we don't have a generic broadlink component that holds the HOST Info and the dependency and which shares the connection credentials (and maybe the connection) with the platforms.